src/auro-toast.js

```diff
@@ -131,6 +132,10 @@ export class AuroToast extends LitElement {
       },
       noIcon: {
         type: Boolean
+      },
+      disableAutoHide: {
```

Comment:
default value missing in constructor; for all properties in this component.

Reply:
We don't want defaults on this properties (even `false`) on this attributes. Having a value of any kind will cause the reflected attributes to appear in the DOM and force style and functional behavior that shouldn't occur until after a value is set. (e.g. applying visible=false will put that into the DOM and the presence of the visible attribute at all forces the toast to display.)